[2022-10-06T12:42:20.754047+00:00] deprecation.INFO: User Deprecated: The "Monolog\Logger" class is considered final. It may change without further notice as of its next major version. You should not extend it from "Symfony\Bridge\Monolog\Logger". {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"Monolog\\Logger\" class is considered final. It may change without further notice as of its next major version. You should not extend it from \"Symfony\\Bridge\\Monolog\\Logger\". at /var/www/html/vendor/symfony/error-handler/DebugClassLoader.php:331)"} []
[2022-10-06T12:50:40.955363+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:1234/"" at /var/www/html/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:1234/\" at /var/www/html/vendor/symfony/http-kernel/EventListener/RouterListener.php:128)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /var/www/html/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:185)"} []
[2022-10-06T12:50:45.384017+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"2d2445"},"request_uri":"http://localhost:1234/_wdt/2d2445","method":"GET"} []
[2022-10-06T13:06:58.197528+00:00] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"2d2445"},"request_uri":"http://localhost:1234/_profiler/2d2445?panel=request","method":"GET"} []
[2022-10-06T13:07:01.465453+00:00] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"2d2445"},"request_uri":"http://localhost:1234/_profiler/2d2445?panel=logger","method":"GET"} []
[2022-10-06T13:07:01.468894+00:00] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"2d2445"},"request_uri":"http://localhost:1234/_profiler/2d2445?panel=logger","method":"GET"} []
[2022-10-06T13:07:01.469076+00:00] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"2d2445"},"request_uri":"http://localhost:1234/_profiler/2d2445?panel=logger","method":"GET"} []
[2022-10-06T13:07:02.391896+00:00] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"2d2445"},"request_uri":"http://localhost:1234/_profiler/2d2445?panel=logger","method":"GET"} []
[2022-10-06T13:07:02.997392+00:00] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"2d2445"},"request_uri":"http://localhost:1234/_profiler/2d2445?panel=logger","method":"GET"} []
[2022-10-06T13:19:24.512043+00:00] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"2d2445"},"request_uri":"http://localhost:1234/_profiler/2d2445?panel=logger","method":"GET"} []
[2022-10-06T13:23:28.132110+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:1234/"" at /var/www/html/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:1234/\" at /var/www/html/vendor/symfony/http-kernel/EventListener/RouterListener.php:128)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /var/www/html/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:185)"} []
[2022-10-06T13:23:32.000135+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"f12e8f"},"request_uri":"http://localhost:1234/_wdt/f12e8f","method":"GET"} []
[2022-10-06T13:29:47.156499+00:00] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"f12e8f"},"request_uri":"http://localhost:1234/_profiler/f12e8f?panel=request","method":"GET"} []
[2022-10-06T14:47:42.549592+00:00] deprecation.INFO: User Deprecated: The "Monolog\Logger" class is considered final. It may change without further notice as of its next major version. You should not extend it from "Symfony\Bridge\Monolog\Logger". {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"Monolog\\Logger\" class is considered final. It may change without further notice as of its next major version. You should not extend it from \"Symfony\\Bridge\\Monolog\\Logger\". at /var/www/html/vendor/symfony/error-handler/DebugClassLoader.php:331)"} []
[2022-10-06T14:47:43.283623+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_mysql","host":"db","port":3306,"user":"root","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"symfony_db","serverVersion":"mariadb-10.7.1","charset":"utf8mb4"} {"params":{"url":"<redacted>","driver":"pdo_mysql","host":"db","port":3306,"user":"root","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"symfony_db","serverVersion":"mariadb-10.7.1","charset":"utf8mb4"}} []
[2022-10-06T14:47:43.610944+00:00] console.CRITICAL: Error thrown while running command "make:migration". Message: "An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'symfony_db'" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 1049): An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'symfony_db' at /var/www/html/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:101)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'symfony_db' at /var/www/html/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:28)\n[previous exception] [object] (PDOException(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'symfony_db' at /var/www/html/vendor/doctrine/dbal/src/Driver/PDO/MySQL/Driver.php:28)","command":"make:migration","message":"An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'symfony_db'"} []
[2022-10-06T14:47:43.618725+00:00] console.DEBUG: Command "make:migration" exited with code "1049" {"command":"make:migration","code":1049} []
[2022-10-06T14:50:48.232120+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_mysql","host":"db","port":3306,"user":"root","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"db","serverVersion":"mariadb-10.7.1","charset":"utf8mb4"} {"params":{"url":"<redacted>","driver":"pdo_mysql","host":"db","port":3306,"user":"root","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"db","serverVersion":"mariadb-10.7.1","charset":"utf8mb4"}} []
[2022-10-06T14:50:48.555019+00:00] console.CRITICAL: Error thrown while running command "make:migration". Message: "An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'db'" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 1049): An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'db' at /var/www/html/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:101)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'db' at /var/www/html/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:28)\n[previous exception] [object] (PDOException(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'db' at /var/www/html/vendor/doctrine/dbal/src/Driver/PDO/MySQL/Driver.php:28)","command":"make:migration","message":"An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'db'"} []
[2022-10-06T14:50:48.568676+00:00] console.DEBUG: Command "make:migration" exited with code "1049" {"command":"make:migration","code":1049} []
[2022-10-06T14:52:19.151841+00:00] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"f12e8f"},"request_uri":"http://localhost:1234/_profiler/f12e8f?panel=request","method":"GET"} []
[2022-10-06T14:52:56.294707+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_mysql","host":"db","port":3306,"user":"root","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"db-data","serverVersion":"mariadb-10.7.1","charset":"utf8mb4"} {"params":{"url":"<redacted>","driver":"pdo_mysql","host":"db","port":3306,"user":"root","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"db-data","serverVersion":"mariadb-10.7.1","charset":"utf8mb4"}} []
[2022-10-06T14:52:56.472248+00:00] console.CRITICAL: Error thrown while running command "make:migration". Message: "An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'db-data'" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 1049): An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'db-data' at /var/www/html/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:101)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'db-data' at /var/www/html/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:28)\n[previous exception] [object] (PDOException(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'db-data' at /var/www/html/vendor/doctrine/dbal/src/Driver/PDO/MySQL/Driver.php:28)","command":"make:migration","message":"An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'db-data'"} []
[2022-10-06T14:52:56.480331+00:00] console.DEBUG: Command "make:migration" exited with code "1049" {"command":"make:migration","code":1049} []
[2022-10-06T14:54:06.049531+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_mysql","host":"db","port":3306,"user":"root","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"root","serverVersion":"mariadb-10.7.1","charset":"utf8mb4"} {"params":{"url":"<redacted>","driver":"pdo_mysql","host":"db","port":3306,"user":"root","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"root","serverVersion":"mariadb-10.7.1","charset":"utf8mb4"}} []
[2022-10-06T14:54:06.207639+00:00] console.CRITICAL: Error thrown while running command "make:migration". Message: "An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'root'" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 1049): An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'root' at /var/www/html/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:101)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'root' at /var/www/html/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:28)\n[previous exception] [object] (PDOException(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'root' at /var/www/html/vendor/doctrine/dbal/src/Driver/PDO/MySQL/Driver.php:28)","command":"make:migration","message":"An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'root'"} []
[2022-10-06T14:54:06.214238+00:00] console.DEBUG: Command "make:migration" exited with code "1049" {"command":"make:migration","code":1049} []
[2022-10-06T14:54:41.661458+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_mysql","host":"db","port":3306,"user":"root","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"app","serverVersion":"mariadb-10.7.1","charset":"utf8mb4"} {"params":{"url":"<redacted>","driver":"pdo_mysql","host":"db","port":3306,"user":"root","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"app","serverVersion":"mariadb-10.7.1","charset":"utf8mb4"}} []
[2022-10-06T14:54:41.818206+00:00] console.CRITICAL: Error thrown while running command "make:migration". Message: "An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'app'" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 1049): An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'app' at /var/www/html/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:101)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'app' at /var/www/html/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:28)\n[previous exception] [object] (PDOException(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'app' at /var/www/html/vendor/doctrine/dbal/src/Driver/PDO/MySQL/Driver.php:28)","command":"make:migration","message":"An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'app'"} []
[2022-10-06T14:54:41.825470+00:00] console.DEBUG: Command "make:migration" exited with code "1049" {"command":"make:migration","code":1049} []
[2022-10-06T15:02:56.836069+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_mysql","host":"db","port":3306,"user":"root","password":"<redacted>","driverOptions":[],"serverVersion":"mariadb-10.7.1","defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"master","charset":"utf8mb4"} {"params":{"url":"<redacted>","driver":"pdo_mysql","host":"db","port":3306,"user":"root","password":"<redacted>","driverOptions":[],"serverVersion":"mariadb-10.7.1","defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"master","charset":"utf8mb4"}} []
[2022-10-06T15:02:57.124454+00:00] console.CRITICAL: Error thrown while running command "make:migration". Message: "An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'master'" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 1049): An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'master' at /var/www/html/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:101)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'master' at /var/www/html/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:28)\n[previous exception] [object] (PDOException(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'master' at /var/www/html/vendor/doctrine/dbal/src/Driver/PDO/MySQL/Driver.php:28)","command":"make:migration","message":"An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'master'"} []
[2022-10-06T15:02:57.136847+00:00] console.DEBUG: Command "make:migration" exited with code "1049" {"command":"make:migration","code":1049} []
[2022-10-06T15:03:22.192971+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_mysql","host":"db","port":3306,"user":"root","password":"<redacted>","driverOptions":[],"serverVersion":"mariadb-10.7.1","defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"app","charset":"utf8mb4"} {"params":{"url":"<redacted>","driver":"pdo_mysql","host":"db","port":3306,"user":"root","password":"<redacted>","driverOptions":[],"serverVersion":"mariadb-10.7.1","defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"app","charset":"utf8mb4"}} []
[2022-10-06T15:03:22.382520+00:00] console.CRITICAL: Error thrown while running command "make:migration". Message: "An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'app'" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 1049): An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'app' at /var/www/html/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:101)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'app' at /var/www/html/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:28)\n[previous exception] [object] (PDOException(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'app' at /var/www/html/vendor/doctrine/dbal/src/Driver/PDO/MySQL/Driver.php:28)","command":"make:migration","message":"An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'app'"} []
[2022-10-06T15:03:22.391687+00:00] console.DEBUG: Command "make:migration" exited with code "1049" {"command":"make:migration","code":1049} []
[2022-10-06T15:05:59.311012+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_mysql","host":"db","port":3306,"user":"root","password":"<redacted>","driverOptions":[],"serverVersion":"mariadb-10.7.1","defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"symfony_db","charset":"utf8mb4"} {"params":{"url":"<redacted>","driver":"pdo_mysql","host":"db","port":3306,"user":"root","password":"<redacted>","driverOptions":[],"serverVersion":"mariadb-10.7.1","defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"symfony_db","charset":"utf8mb4"}} []
[2022-10-06T15:05:59.555100+00:00] console.CRITICAL: Error thrown while running command "make:migration". Message: "An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'symfony_db'" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 1049): An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'symfony_db' at /var/www/html/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:101)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'symfony_db' at /var/www/html/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:28)\n[previous exception] [object] (PDOException(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'symfony_db' at /var/www/html/vendor/doctrine/dbal/src/Driver/PDO/MySQL/Driver.php:28)","command":"make:migration","message":"An exception occurred in the driver: SQLSTATE[HY000] [1049] Unknown database 'symfony_db'"} []
[2022-10-06T15:05:59.562916+00:00] console.DEBUG: Command "make:migration" exited with code "1049" {"command":"make:migration","code":1049} []
